packages/api/tsconfig.json

```
@@ -7,6 +7,6 @@
   "include": ["./src"],
   "references": [
     { "path": "../xrpc/tsconfig.build.json" },
-    { "path": "../xrpc-cli/tsconfig.build.json" },
+    { "path": "../xrpc-server/tsconfig.build.json" }
```

I was seeing an error here in my IDE coming from typescript.  I _think_ this is the right change (xrpc-cli no longer exists), but if not feel free to tweak it or I can revert this change.

---

Ahh got it, that's clear 👍 
```suggestion
    { "path": "../lex-cli/tsconfig.build.json" }
```